src/diffusers/loaders/lora.py

```diff
@@ -1632,6 +1634,12 @@ def pack_weights(layers, prefix):
         if transformer_lora_layers:
             state_dict.update(pack_weights(transformer_lora_layers, cls.transformer_name))
 
+        if text_encoder_lora_layers:
+            state_dict.update(pack_weights(text_encoder_lora_layers, "text_encoder"))
+
+        if text_encoder_2_lora_layers:
+            state_dict.update(pack_weights(text_encoder_2_lora_layers, "text_encoder_2"))
```

Comment:
Can we confirm this via experiments that text encoder 3 training doesn't matter too much? Can be done separately and won't block this PR.

Reply:
Actually we skipped not because we think it doesn't matter but because we thought dealing with training the T5 would be a different animal than the already well known CLIP text encoder training (also on VRAM consumption side). So indeed we left it to a future PR to investigate the T5 training!

Reply:
IMO we don’t need to be urgent in this case. We can allow ourselves the time we need for training all the text encoders and have a full fledged text encoder training support.

Reply:
Slightly worried about the dynamics about this so, let’s make sure we run ample experiments to see if training two text encoders while keeping the other one fixed works as expected.